docusaurus/docs/user-docs/plugins/strapi-plugins.md

```diff
@@ -10,7 +10,7 @@ Strapi builds and maintains plugins that extend the functionality of a core Stra
 :::note
 
 - Some Strapi Starters and Templates might install additional plugins beyond the default plugins listed below.
-- Some plugin options are only available with an [Enterprise edition license](https://strapi.io/pricing-self-hosted), and are marked with  <EnterpriseBadge link="https://strapi.io/pricing-self-hosted"/> in this reference guide.
+- If plugin options are only available with an [Enterprise edition license](https://strapi.io/pricing-self-hosted), they are marked with  <EnterpriseBadge link="https://strapi.io/pricing-self-hosted"/> in this reference guide.
```

Comment:
```suggestion
- If plugin options are only available with an [Enterprise edition license](https://strapi.io/pricing-self-hosted), they are marked with  <EnterpriseBadge /> in this reference guide.
```

We can even remove the `link` attribute here, as the passed value is the same as the [default one](https://github.com/strapi/documentation/blob/main/docusaurus/src/components/EnterpriseBadge.js#L4) 🙂